textproc/recoll/files/patch-recoll-pro-stdlib.diff

```diff
@@ -0,0 +1,12 @@
+diff --git a/src/qtgui/recoll.pro.in b/src/qtgui/recoll.pro.in
+index f69aaa27..218c89b2 100644
+--- qtgui/recoll.pro.in
++++ qtgui/recoll.pro.in
+@@ -148,6 +148,7 @@ contains( UNAME, SunOS ) {
+ 
+ macx {
+     ICON = images/recoll.icns
++    LIBS += -stdlib=libc++
```

Comment:
This may not be hardcoded. You should use something like
```suggestion
+    LIBS += -stdlib=@@STDLIB@@
```
and then reinplace in the Portfile.

But more importantly the compiler being used in wrong, and I'm not sure if it's this port or our general Qt support that is to blame for it.

Reply:
I tried the change, but @@STDLIB@@ does not get substituted. Actually I don't understand the double @@ and, in any case, wouldn't this suppose that the configure script finds out about STDLIB and substitutes it, (which it does not do currently) ? Please explain so that I can complete the fix.

Reply:
Take a look at [ace](https://github.com/macports/macports-ports/tree/master/devel/ace/) or [root6](https://github.com/macports/macports-ports/tree/master/science/root6/) or [py-protobuf3](https://github.com/macports/macports-ports/tree/master/python/py-protobuf3).

You need to add `reinplace` to the `Portfile` to do the actual substitution.